src/controller.rs

```diff
@@ -403,14 +403,68 @@ fn pod_template(config: &WasmCloudHostConfig, _ctx: Arc<Context>) -> PodTemplate
         ..Default::default()
     }];
 
-    let mut volume_mounts = vec![VolumeMount {
+    let mut nats_volume_mounts = vec![VolumeMount {
         name: "nats-config".to_string(),
         mount_path: "/nats/nats.conf".to_string(),
         read_only: Some(true),
         sub_path: Some("nats.conf".to_string()),
         ..Default::default()
     }];
 
+    let mut wasm_volume_mounts: Vec<VolumeMount> = vec![];
+    if let Some(authorities) = &config
+        .spec
+        .certificates
+        .clone()
+        .and_then(|certificates| certificates.authorities)
+    {
+        for (i, authority) in authorities.iter().enumerate() {
+            // configmaps
+            if let Some(configmap) = &authority.config_map_ref {
+                let volume_name = format!("cert-authority-{i}");
+                let volume_path = format!("/wasmcloud/certificates/authorities/{volume_name}");
+                match discover_configmap_certificates(
+                    config.namespace().unwrap_or_default().as_str(),
+                    configmap.name.as_str(),
```

Comment:
These would be better written as
```suggestion
                    &config.namespace().unwrap_or_default(),
                    &configmap.name,
```